content/docs/esc/faq.md

```diff
@@ -13,36 +13,34 @@ aliases:
   - /docs/pulumi-cloud/esc/faq
 ---
 
-## Is it safe to use Pulumi ESC for my production stacks?
+## Why did Pulumi launch ESC?
 
-Yes, Pulumi ESC is served using the same [highly secure](/security/pulumi-cloud-security-whitepaper.pdf) and highly available infrastructure as the rest of Pulumi Cloud. The service is currently in preview as we are rapidly iterating and improving the feature set. It is unlikely that we will make any breaking changes. Should we introduce a new API, we are dedicated to ensuring optimal compatibility to prevent any interruptions to your infrastructure.
+We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC brings the same software engineering approach to secrets and configuration, allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks. just like Pulumi IaC. It can be used for all applications and infrastructure - with or without Pulumi IaC.
 
-## Why did Pulumi launch ESC?
+## What is the pricing of Pulumi ESC?
+
+See our [pricing page](https://www.pulumi.com/pricing/) for details.
 
-We launched Pulumi ESC in response to customer feedback about their difficulties in managing config and secrets causing sprawl and duplications across stacks. Pulumi ESC is a natural extension of Pulumi IaC by allowing [hierarchical](/docs/esc/#configuration-as-code) configurations that can be used across stacks.
+## What counts as a secret towards pricing?
 
-## When will Pulumi ESC be generally available?
+Secret includes both [static secrets](/docs/esc/get-started/store-and-retrieve-secrets/), [dynamic login credentials](/docs/esc/integrations/dynamic-login-credentials/) and [dynamic secrets](/docs/esc/integrations/dynamic-secrets/).
 
-Pulumi ESC will be generally available in 2024.
+In other words,
 
-## What will be the pricing of Pulumi ESC?
+- When using the Pulumi ESC Document Editor, each definition of fn::secret:* and fn::open::* (except Pulumi-stacks provider) is counted as a secret.
+- When using the Pulumi ESC Table Editor, each secret and provider config type (except Pulumi-stacks) is counted as a secret.
 
-During the preview, the product is free to use. When the product is made generally available, it will have a pricing model that is competitive with the market.
+The number of secrets only from the latest environment revision is counted towards your billing.
```

Comment:
Would reword as "Only the secrets from the latest environment revision are counted towards your billing."